### ISSUE TITLE
Reformatting does not work as expected for INNER JOIN without the optional INNER key word

### ISSUE THREAD

**Issue by kjetil-kilhavn** [SAP/abap-cleaner#221]
ABAP Cleaner version 1.12.0 with default configuration

```
    SELECT
      FROM c_respymgmtteamheadertp AS team_header
      JOIN c_respymgmtteamownertp AS team_owner ON team_owner~respymgmtteamid = team_header~respymgmtteamid
      FIELDS DISTINCT team_owner~respymgmtteamowneruserid
      WHERE team_header~respymgmtteamnameforsearch = @team_name_for_search
        AND team_owner~teamownerenddate   >= @at_date
        AND team_owner~teamownerstartdate <= @at_date
      INTO TABLE @result.
```

Gets reformatted to 

```
    SELECT
      FROM                                                                           c_respymgmtteamheadertp AS team_header
      JOIN c_respymgmtteamownertp
           AS team_owner ON team_owner~respymgmtteamid = team_header~respymgmtteamid
      FIELDS DISTINCT team_owner~respymgmtteamowneruserid
      WHERE team_header~respymgmtteamnameforsearch = @team_name_for_search
        AND team_owner~teamownerenddate   >= @at_date
        AND team_owner~teamownerstartdate <= @at_date
      INTO TABLE @result.

```

Whereas if I add the optional INNER key word it works as expected

```
    SELECT
      FROM c_respymgmtteamheadertp AS team_header
             INNER JOIN
               c_respymgmtteamownertp AS team_owner ON team_owner~respymgmtteamid = team_header~respymgmtteamid
      FIELDS DISTINCT team_owner~respymgmtteamowneruserid
      WHERE team_header~respymgmtteamnameforsearch = @team_name_for_search
        AND team_owner~teamownerenddate   >= @at_date 
        AND team_owner~teamownerstartdate <= @at_date
      INTO TABLE @result.
```

**Comment by jmgrassau** [comment]
Hi Kjetil, 

thank you very much for reporting this – I indeed overlooked that 'INNER' is optional! This will be fixed with the next release. 

![image](https://github.com/SAP/abap-cleaner/assets/79639298/7f5d36ec-de67-4198-981e-7461c01a2958)

Kind regards,
Jörg-Michael

P.S.: I really like "1983 Commodore 64" in your profile – same here! :-)

**Comment by jmgrassau** [comment]
Hi Kjetil, 

thanks again for reporting this! This bug should now be fixed in version 1.13.0, which was just released.

Kind regards,
Jörg-Michael
